docs/tutorial/part-zero/index.md

```diff
@@ -1,194 +1,194 @@
 ---
-title: Set Up Your Development Environment
+title: Menyiapkan Lingkungan Pengembangan Anda
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Before you start building your first Gatsby site, you’ll need to familiarize yourself with some core web technologies and make sure that you have installed all required software tools.
+Sebelum memulai membangun situs Gatsby pertama Anda, Anda harus membiasakan diri dengan beberapa teknologi web utama dan memastikan bahwa Anda telah menginstal semua perangkat lunak yang diperlukan.
 
-## Familiarize yourself with the command line
+## Biasakan diri Anda dengan baris perintah
 
-The command line is a text-based interface used to run commands on your computer. You’ll also often see it referred to as the terminal. In this tutorial, we’ll use both interchangeably. It’s a lot like using the Finder on a Mac or Explorer on Windows. Finder and Explorer are examples of graphical user interfaces (GUI). The command line is a powerful, text-based way to interact with your computer.
+Baris perintah adalah sebuah antarmuka berbasis teks yang digunakan untuk menjalankan perintah di komputer Anda. Baris perintah juga sering kita ketahui sebagai terminal. Dalam tutorial ini, kita akan menggunakan keduanya secara bergantian. Baris perintah ini sangat mirip seperti kita menggunakan Finder di Mac atau Explorer di Windows. Finder dan Explorer adalah contoh antarmuka pengguna grafis (GUI). Baris perintah adalah cara yang ampuh untuk berinteraksi dengan komputer Anda dengan berbasis teks.
```

Comment:
I think it will be good to keep "command line" with italic text rather than "baris perintah"